spark-atlas-connector/src/test/scala/com/hortonworks/spark/atlas/sql/CreateDataSourceTableAsSelectHarvesterSuite.scala

```diff
@@ -28,7 +30,8 @@ import org.apache.spark.sql.execution.datasources.DataSource
 import org.apache.spark.sql.types.StructType
 import org.scalatest.{FunSuite, Matchers}
 
-
+// This is not leveraging BaseHarvesterSuite, as it doesn't need to be tested with
+// both non-remote HMS and remote HMS cases.
```

Comment:
This actually tests whether SAC fills attributes for Spark table entity properly, so no need to be tested with remote HMS cases.